implementations/rust/ockam/ockam_command/src/ockam_run.rs

```diff
@@ -0,0 +1,46 @@
+use clap::Parser;
```

Comment:
The `ockam_command` and `ockam_run` file names are confusing to me. Just reading the names, I'd say the main file is `ockam_command` since it's named after the crate.

Maybe `ockam_run` could be named `entry_point` and `ockam_command` just `command`. We can discuss it separately and do it in another PR.

Reply:
I agree. I renamed to `entry_point`, `command` and `subcommand` (for `ockam_subcommand`).